test/sanity/code-smell/use-argspec-type-path.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
This file should not be part of the PR. I'm guessing it was accidentally pulled in during a rebase.

Reply:
@mattclay after removing this file this test started to fail again: https://github.com/ansible/ansible/pull/45541#issuecomment-576212862

Reply:
You will need to add an entry to the `test/sanity/ignore.txt` file:

```
lib/ansible/modules/system/launchd.py use-argspec-type-path
```